functional-tests/README.md

```diff
@@ -51,8 +53,35 @@ Only on windows:
 
 Only on MacOS:
 
-    ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
+    DEPRECATED ./gradlew safariTest -DsafariTest.single=CustomJUnitSpecRunner
```

Comment:
Can we remove the entire code block for safari if we're not supporting it at all?
Also why is it deprecated?

Reply:
left a note that gives a todo to fix